src/main/java/com/nha/abdm/wrapper/hrp/mongo/tables/Patients.java

```diff
@@ -1,6 +1,6 @@
 package com.nha.abdm.wrapper.hrp.mongo.tables;
 
-import com.nha.abdm.wrapper.hrp.common.CareContextBuilder;
+import com.nha.abdm.wrapper.hrp.CommonHelpers.CareContextBuilder;
 import lombok.Data;
```

Comment:
Instead of writing the full constructor method, the same can be achieved via the `@AllArgsConstructor` and `@NoArgsConstructor` annotations.

Reply:
Applied @AllArgsConstructor and @NoArgsConstructor annotations